src/appleseed/foundation/math/sampling/qmcsamplingcontext.h

```diff
@@ -279,7 +335,18 @@ template <typename RNG>
 template <typename T>
 inline T QMCSamplingContext<RNG>::next2()
 {
-    return next2(Tag<T>());
+    if (m_cmj_params.m_enabled)
```

Comment:
Can we somehow get m_cmj_params.m_enabled to be true while in RNG sampling mode?

Reply:
No, you can see that in the constructor of CMJParameters